reduction/imaging_parameters.py

```diff
@@ -348,43 +348,77 @@
                               },
     
        #W43-MM2 B6
-    'W43-MM2_B6_12M_robust0': {'threshold': {0: '2.0mJy', 1: '2.0mJy', 2: '0.7mJy', 3: '0.5mJy', 4: 'O.35mJy', 5: '0.25mJy'},
-                               'niter': {0: 1000, 1: 3000, 2: 10000, 3: 12000, 4: 15000, 5: 15000},
-                               'scales': [0,3,9,27],
+    'W43-MM2_B6_12M_robust0': {'threshold': {0: '2.0mJy', 1: '2.0mJy', 2: '0.7mJy', 3: '0.5mJy', 4: '0.35mJy', 5: '0.25mJy', 'final': '0.35mJy'},
+                               'niter': {0: 1000, 1: 3000, 2: 10000, 3: 12000, 4: 15000, 5: 15000, 'final': 22000},
+                               'scales': {0: [0,3,9,27], 1: [0,3,9,27], 2: [0,3,9,27], 3: [0,3,9,27], 4: [0,3,9,27], 'final': [0,3,9,27]},
```

Comment:
there's no need to expand scales like this if they are all thesame

Reply:
They are not the same, 'final' is using one more scale in most of the region (largest is 81 most of the time vs 27 during selfcal)

Reply:
not in this case: look again